### PR TITLE
namei: add page

### PR DESCRIPTION
**Pull request by Anmol-Singh-Jaggi**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by Anmol-Singh-Jaggi** [comment]
For #2214

File: pages/linux/namei.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# namei
+
+> Follows a pathname (which can be a symbolic link) until a terminal point is found (a file/directory/char device etc).
+> This program is useful for finding "too many levels of symbolic links" problems.
+
+- Resolve the pathnames specified as the argument parameters:
+
+`namei {{/path/to/a}} {{/path/to/b}} {{/path/to/c}}`
```
Comment (pxgamer) [inline pages/linux/namei.md]: Remove the prefixing slash from each path (i.e. `{{path/to/...}}`)

File: pages/linux/namei.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display the results in a long-listing format:
+
+`namei --long {{/path/to/a}} {{/path/to/b}} {{/path/to/c}}`
```
Comment (pxgamer) [inline pages/linux/namei.md]: Remove the prefixing slash from each path

File: pages/linux/namei.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show the mode bits of each file type in the style of `ls`:
+
+`namei --modes {{/path/to/a}} {{/path/to/b}} {{/path/to/c}}`
```
Comment (pxgamer) [inline pages/linux/namei.md]: Remove the prefixing slash from each path

File: pages/linux/namei.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show owner and group name of each file:
+
+`namei --owners {{/path/to/a}} {{/path/to/b}} {{/path/to/c}}`
```
Comment (pxgamer) [inline pages/linux/namei.md]: Remove the prefixing slash from each path

File: pages/linux/namei.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Don't follow symlinks while resolving:
+
+`namei --nosymlinks {{/path/to/a}} {{/path/to/b}} {{/path/to/c}}`
```
Comment (pxgamer) [inline pages/linux/namei.md]: Remove the prefixing slash from each path

File: pages/linux/namei.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# namei
+
+> Follows a pathname (which can be a symbolic link) until a terminal point is found (a file/directory/char device etc).
```
Comment (pxgamer) [inline pages/linux/namei.md]: Does it need to mention that a symbolic link can be used? 🤔

I'd have assumed `pathname` would include symlinks.

Comment (Anmol-Singh-Jaggi) [reply pages/linux/namei.md]: Umm but when I first read it, I was a bit confused and had to read the full man page.
Or maybe I am just stupid.

File: pages/linux/namei.md (hunk 5 above)
Comment (Anmol-Singh-Jaggi) [reply pages/linux/namei.md]: Nice catch. Done!

File: pages/linux/namei.md (hunk 6 above)
Comment (pxgamer) [reply pages/linux/namei.md]: Fair enough 🙂 Well, looks good to me then!

**Pull request merged**